azerothcore/azerothcore-wotlk
# fix(DB/Gameobject): Add (more) Tailasher Eggs

## Changes Proposed:
-  Add new spawn location for Tailasher Egg's (3240 object id)



## Issues Addressed:
- Closes #5343



## SOURCE:

https://classic.wowhead.com/quest=815/break-a-few-eggs

## Tests Performed:

- Tested on Windows 10 on my local game server.



## How to Test the Changes:

.go zonexy 68.2 88.6 14 to see one of the new spawn locations, please see the original bugtracker for more coordinates to test, or compare with wowheads map of locations for the tailasher eggs. Beware that some of the locations on wowhead are inaccurate and I have decided to not include them.

## Known Issues and TODO List:

N/A


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
